crypto/keyring/keyring.go

```diff
@@ -292,8 +308,10 @@ func (ks keystore) ImportPrivKey(uid, armor, passphrase string) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to decrypt private key")
 	}
-
-	_, err = ks.writeLocalKey(uid, privKey, hd.PubKeyType(algo))
+	// TODO fix PubKeyType
+	//pubKeyTYpe has only only Local right? ke.GetLocal().PubKeyType
+	var i SignatureAlgo = algo
+	_, err = ks.writeLocalKey(uid, privKey, i)
```

Comment:
should i pass algo as string as third argument or algo as Signaturealgo interface in writeLocalKey?